prac_05/wimbledon.py

```diff
@@ -0,0 +1,50 @@
+"""
+Wimbledon
+Estimate: 40 min
+Actual: 1.5 hour
+"""
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    with open(FILENAME, 'r', encoding="utf-8-sig") as in_file:
+        champions_to_amount_of_wins = process_data(in_file, "Champion")
+        countries_to_amount_of_wins = process_data(in_file, "Country")
+        print_champions(champions_to_amount_of_wins)
+        print_countries(countries_to_amount_of_wins)
+
+
+def process_data(file, column_name):
+    """This function will process the data"""
+    key_to_amount = {}
+    info_line = file.readline()
+    info_line = info_line.split(",")
+    index = info_line.index(column_name)
+    for line in file:
+        champion = line.split(",")[index]
```

Comment:
For this data champion refers to the person that won, but you use this as a variable name for both the Champion and the Country. Does it still make sense in the Country context? For example, the `champion` wouldn't be AUS, it would be Rod Laver.